documentation/geo-search.html

```diff
@@ -38,7 +38,7 @@
 <h2 id="restrict">Restrict</h2>
 <p>
 The preferred API for adding a geographical restriction is to use a
-<a href="reference/query-language-reference.html#geolocation">geoLocation</a> clause in the YQL statement, example:
+<a href="reference/query-language-reference.html#geoLocation">geoLocation</a> clause in the YQL statement, example:
```

Comment:
we try to keep the anchors all lowercase, so should be changed to lower everywhere. ok in next pr